projects/hipblaslt/tensilelite/Tensile/TensileMergeLibrary.py

```diff
@@ -327,9 +328,14 @@ def avoidRegressions(originalDir, incrementalDir, outputPath, forceMerge, noEff=
         # So far "SolutionIndex" in logic yamls has zero impact on actual 1-1 size mapping (but the order of the Solution does)
         # since mergeLogic() takes that value very seriously so we reindex them here so it doesn't choke on duplicated SolutionIndex
         oriData, numRemoved, numSolutions, numKernels = removeDuplicatedSolutions(oriData)
+        if oriData[7] is None:
+            oriData[7] = []
         msg("Base logic file:", numRemoved, "duplicated solution(s) removed,",\
             "sizes: %d, solutions: %d, kernels: %d"%(len(oriData[7]),numSolutions,numKernels))
+        
         incData, numRemoved, numSolutions, numKernels = removeDuplicatedSolutions(incData)
+        if incData[7] is None:
+            incData[7] = []
         msg("Inc logic file:", numRemoved, "duplicated solution(s) removed,",\
             "sizes: %d, solutions: %d, kernels: %d"%(len(incData[7]),numSolutions,numKernels))
```

Comment:
```suggestion
        oriData, numRemoved, numSolutions, numKernels = removeDuplicatedSolutions(oriData)
        msg("Base logic file:", numRemoved, "duplicated solution(s) removed,",\
            "sizes: %d, solutions: %d, kernels: %d"%(len(oriData[7] or []),numSolutions,numKernels))
        incData, numRemoved, numSolutions, numKernels = removeDuplicatedSolutions(incData)
        msg("Inc logic file:", numRemoved, "duplicated solution(s) removed,",\
            "sizes: %d, solutions: %d, kernels: %d"%(len(incData[7] or []),numSolutions,numKernels))
```

I should probably do this to avoid overwriting the yaml file.

and line 218 should be changed to following, to avoid error
```
    origNumSizes = len(oriData[7] or [])
```